tools/dgeni/src/transforms/daffodil-guides-package/reader/guide-file.reader.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { extractTitle } from './guide-file.reader';
+
+describe('guideFileReader', () => {
+	
+	describe('extractTitle', () => {
```

Comment:
Can you add a test covering the result of toc?

Reply:
I don't think we should test the results of external packages.

Reply:
I think it's ok as long as we test only the API surface. If `toc` were to change, I would like to know if it violates our expectations.

Reply:
added